crates/lumina_server/src/lobby/in_game.rs

```diff
@@ -75,10 +83,31 @@ fn init_spaceship_position(
             continue;
         }
 
-        *position = Position(desired_position);
-        commands.entity(entity).insert(PositionInitialized);
+        let (spawn_position, team_type) = if team_toggle.toggle {
+            (bottom_left, TeamType::A)
+        } else {
+            (upper_right, TeamType::B)
+        };
+
+        team_toggle.toggle = !team_toggle.toggle; // Toggle the team for the next spawn
+
+        *position = Position(spawn_position);
+        commands
+            .entity(entity)
+            .insert((PositionInitialized, team_type));
     }
 }
 
 #[derive(Component)]
 pub struct PositionInitialized;
+
+#[derive(Resource)]
+pub struct TeamToggle {
+    pub toggle: bool,
+}
+
+impl TeamToggle {
+    pub fn new() -> Self {
+        Self { toggle: true }
+    }
+}
```

Comment:
```suggestion
```

This is redundant.

Reply:
Insert the `TeamType` component into the spaceship entities in `spawn_point.rs` and then query it from the `init_spaceship` system and use that.